tests/test_D0_modelling_and_optimization.py

```diff
@@ -181,7 +181,8 @@ def test_error_raise_MVSOemofError_if_solver_could_not_finish_simulation(margs):
 def setup_function():
     if os.path.exists(TEST_OUTPUT_PATH):
         shutil.rmtree(TEST_OUTPUT_PATH, ignore_errors=True, onerror=None)
-    os.mkdir(TEST_OUTPUT_PATH)
+    if not os.path.exists(TEST_OUTPUT_PATH):
```

Comment:
```suggestion
    if os.path.exists(TEST_OUTPUT_PATH) is False:
```

Alternative explicit formulation